<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="artur-petin" ignored="0" name="Command line test" tests="28" failures="11" timestamp="15 янв 2018 11:19:48 GMT" time="966.721" errors="0">
  <testcase name="@BeforeMethod setUp" time="4.576" classname="ru.stqa.pft.statr.tests.TestBase">
    <failure type="org.openqa.selenium.WebDriverException" message="unknown error: Element &amp;lt;button ng-disabled=&amp;quot;user.user.length &amp;lt;= 0 || user.pwd.length &amp;lt;= 0 || logining&amp;quot; style=&amp;quot;margin-right: 5px;&amp;quot; type=&amp;quot;submit&amp;quot; id=&amp;quot;btn-login&amp;quot; href=&amp;quot;#&amp;quot; class=&amp;quot;btn btn-success pull-right&amp;quot;&amp;gt;...&amp;lt;/button&amp;gt; is not clickable at point (1346, 266). Other element would receive the click: &amp;lt;form id=&amp;quot;loginform&amp;quot; class=&amp;quot;form-horizontal ng-valid ng-dirty ng-valid-parse&amp;quot; role=&amp;quot;form&amp;quot; ng-submit=&amp;quot;submit()&amp;quot;&amp;gt;...&amp;lt;/form&amp;gt;
  (Session info: chrome=57.0.2987.98)
  (Driver info: chromedriver=2.28.455520 (cc17746adff54984afff480136733114c6b3704b),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 87 milliseconds
Build info: version: &amp;apos;3.4.0&amp;apos;, revision: &amp;apos;unknown&amp;apos;, time: &amp;apos;unknown&amp;apos;
System info: host: &amp;apos;ARTUR-PETIN&amp;apos;, ip: &amp;apos;192.168.5.182&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0_101&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.28.455520 (cc17746adff54984afff480136733114c6b3704b), userDataDir=C:\Users\ARTUR~1.PET\AppData\Local\Temp\scoped_dir4684_30166}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=57.0.2987.98, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 1a33e46b6acb2e286bbe87b3c13aae5a">
      <![CDATA[org.openqa.selenium.WebDriverException: unknown error: Element <button ng-disabled="user.user.length <= 0 || user.pwd.length <= 0 || logining" style="margin-right: 5px;" type="submit" id="btn-login" href="#" class="btn btn-success pull-right">...</button> is not clickable at point (1346, 266). Other element would receive the click: <form id="loginform" class="form-horizontal ng-valid ng-dirty ng-valid-parse" role="form" ng-submit="submit()">...</form>
  (Session info: chrome=57.0.2987.98)
  (Driver info: chromedriver=2.28.455520 (cc17746adff54984afff480136733114c6b3704b),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 87 milliseconds
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'ARTUR-PETIN', ip: '192.168.5.182', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_101'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.28.455520 (cc17746adff54984afff480136733114c6b3704b), userDataDir=C:\Users\ARTUR~1.PET\AppData\Local\Temp\scoped_dir4684_30166}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=57.0.2987.98, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 1a33e46b6acb2e286bbe87b3c13aae5a
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:215)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:167)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:671)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:272)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:82)
	at ru.stqa.pft.statr.appmanager.ApplicationManager.login(ApplicationManager.java:96)
	at ru.stqa.pft.statr.appmanager.ApplicationManager.init(ApplicationManager.java:85)
	at ru.stqa.pft.statr.tests.TestBase.setUp(TestBase.java:18)
... Removed 41 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="ru.stqa.pft.statr.tests.TestBase">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="testCurrentSituation" time="3.151" classname="ru.stqa.pft.statr.tests.TestCurrentSituation">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at ru.stqa.pft.statr.appmanager.ApplicationManager.vereficryCurrentSituatuin(ApplicationManager.java:109)
	at ru.stqa.pft.statr.tests.TestCurrentSituation.testCurrentSituation(TestCurrentSituation.java:13)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- testCurrentSituation -->
  <testcase name="testOpen" time="46.261" classname="ru.stqa.pft.statr.tests.Form.TestForms">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.goToFormPage(ApplicationManager.java:136)
	at ru.stqa.pft.statr.tests.Form.TestForms.testOpen(TestForms.java:11)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testOpen -->
  <testcase name="testDannue" time="2.075" classname="ru.stqa.pft.statr.tests.testforms.TestDannue"/>
  <testcase name="testGenDocument" time="454.145" classname="ru.stqa.pft.statr.tests.GenDocument.TestGenDocument">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.goToGenDocument(ApplicationManager.java:144)
	at ru.stqa.pft.statr.tests.GenDocument.TestGenDocument.testGenDocument(TestGenDocument.java:13)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testGenDocument -->
  <testcase name="testStatistics" time="15.536" classname="ru.stqa.pft.statr.tests.testforms.TestEight"/>
  <testcase name="testGenReports" time="21.926" classname="ru.stqa.pft.statr.tests.GenReports.TestGenReports">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.goToGenReports(ApplicationManager.java:667)
	at ru.stqa.pft.statr.tests.GenReports.TestGenReports.testGenReports(TestGenReports.java:9)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testGenReports -->
  <testcase name="testStatistics" time="38.131" classname="ru.stqa.pft.statr.tests.testforms.TestEighteen">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.proverka280(ApplicationManager.java:476)
	at ru.stqa.pft.statr.tests.testforms.TestEighteen.testStatistics(TestEighteen.java:14)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testStatistics -->
  <testcase name="testReference" time="44.953" classname="ru.stqa.pft.statr.tests.Reference.TestReference">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.goToReference(ApplicationManager.java:121)
	at ru.stqa.pft.statr.tests.Reference.TestReference.testReference(TestReference.java:12)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testReference -->
  <testcase name="testStatistics" time="90.619" classname="ru.stqa.pft.statr.tests.testforms.TestEleven">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.proverka244(ApplicationManager.java:356)
	at ru.stqa.pft.statr.tests.testforms.TestEleven.testStatistics(TestEleven.java:14)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatus" time="10.685" classname="ru.stqa.pft.statr.tests.status.TestStatus">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.goToStatus(ApplicationManager.java:118)
	at ru.stqa.pft.statr.tests.status.TestStatus.testStatus(TestStatus.java:13)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testStatus -->
  <testcase name="testStatistics" time="93.25" classname="ru.stqa.pft.statr.tests.testforms.TestFifteen">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.proverka251(ApplicationManager.java:428)
	at ru.stqa.pft.statr.tests.testforms.TestFifteen.testStatistics(TestFifteen.java:14)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="26.511" classname="ru.stqa.pft.statr.tests.testforms.TestFive">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.proverka070(ApplicationManager.java:260)
	at ru.stqa.pft.statr.tests.testforms.TestFive.testStatistics(TestFive.java:14)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="36.908" classname="ru.stqa.pft.statr.tests.testforms.TestFour">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ru.stqa.pft.statr.appmanager.ApplicationManager.proverka064(ApplicationManager.java:244)
	at ru.stqa.pft.statr.tests.testforms.TestFour.testStatistics(TestFour.java:14)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestFourteen">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestNine">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestOne">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestSeven">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestSix">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestSixteen">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestTen">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestThirteen">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestThree">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestTwelve">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testStatistics" time="0.0" classname="ru.stqa.pft.statr.tests.testforms.TestTwo">
    <skipped/>
  </testcase> <!-- testStatistics -->
  <testcase name="testLogs" time="0.0" classname="ru.stqa.pft.statr.tests.TestLogs">
    <skipped/>
  </testcase> <!-- testLogs -->
  <testcase name="testReportGenerator" time="0.0" classname="ru.stqa.pft.statr.tests.TestReportGenerator">
    <skipped/>
  </testcase> <!-- testReportGenerator -->
  <testcase name="testUsers" time="0.0" classname="ru.stqa.pft.statr.tests.testuser.TestUsers">
    <skipped/>
  </testcase> <!-- testUsers -->
  <testcase name="testUserGroup" time="0.0" classname="ru.stqa.pft.statr.tests.usergroup.TestUserGroup">
    <skipped/>
  </testcase> <!-- testUserGroup -->
</testsuite> <!-- Command line test -->
